### ISSUE TITLE
Any support to firebase 9?

### ISSUE THREAD

**Issue by christhoph** [premshree/use-pagination-firestore#10]
I have this error when using the hook with firebase `^9.1.3`:

`Uncaught TypeError: query.limit is not a function`

``` js
import { query, orderBy, collection } from "firebase/firestore";
import { usePagination } from "use-pagination-firestore";

export const useTestPagination = (field, max = 10, order = "desc") => {
  const q = query(
    collection(
      db,
      "test"
    ),
    orderBy(field, order)
  );
  const response = usePagination(q, { limit: max });

  return response;
};
```

Do you know if there is any way to use it without changing the version?

**Comment by premshree** [comment]
Hi @christhoph Firebase 9.x is not currently supported, but I would love to be able to support it. I'll keep this issue open and see if I'm able to get something out next week. Meanwhile, if you have a PR that might help, feel free to help out!

**Comment by christhoph** [comment]
> Hi @christhoph Firebase 9.x is not currently supported, but I would love to be able to support it. I'll keep this issue open and see if I'm able to get something out next week. Meanwhile, if you have a PR that might help, feel free to help out!

I will gladly review it, thanks!

**Comment by rrnara** [comment]
I created a Pull Request here: https://github.com/premshree/use-pagination-firestore/pull/11/files

**Comment by premshree** [comment]
Published [0.6.0](https://www.npmjs.com/package/use-pagination-firestore) – lmk if you run into any issues! Thanks @rrnara
